functions/gyro_straight.py

```diff
@@ -60,21 +60,24 @@ def gyro_straight( left_motor_letter='B', right_motor_letter='A', degrees=9000,
     motor_right = get_motor_by_letter(right_motor_letter)
     # motor_right.preset(0) will reset the relative degrees because otherwise the second time you run this function the relative degrees will start where it left off last time
     motor_right.preset(0)
-    speed, relative_degrees, absolute_degrees, pwm = motor_right.get()
+    motor_left.preset(0)
+    speed, relative_degrees_right, absolute_degrees_right, pwm = motor_right.get()
+    speed, relative_degrees_left, absolute_degrees_left, pwm = motor_right.get()
     #easing stuff.
     pct_degrees = 0
-    print("Start:", speed, relative_degrees, absolute_degrees, pwm)
     motor_pair.start_tank(start_power, start_power)
     #motor stop mode
     while True:
-        speed, relative_degrees, absolute_degrees, pwm = motor_right.get()
+        speed_right, relative_degrees_right, absolute_degrees_right, pwm = motor_right.get()
```

Comment:
can you use pwm_right and left as well here just to be consistent?  it's not good to overwrite a variable

Reply:
got it